test/0_staking.js

```diff
@@ -10,22 +11,53 @@ const expect = require('chai')
     .use(require('chai-bn')(BN))
     .use(require('chai-as-promised'))
     .expect;
-const ValidatorSetContract = require('../utils/getContract')('ValidatorSetAuRa', web3);
-const StakingTokenContract = require('../utils/getContract')('StakingToken', web3);
+const getContract = (((getContractInner, w) => name => getContractInner(name, w))(require('../utils/getContract'), web3));
+const ValidatorSetContract = getContract('ValidatorSetAuRa');
+const StakingTokenContract = getContract('StakingToken');
+const BlockRewardContract = getContract('BlockRewardAuRa');
 const canStakeAndWithdraw = require('../utils/canStakeAndWithdraw');
 const pp = require('../utils/prettyPrint');
 
 describe('Candidates make stakes on themselves', () => {
+    'use strict';
     var minStake;
     var minStakeBN;
     before(async () => {
         minStake = await ValidatorSetContract.instance.methods.getCandidateMinStake().call();
         minStakeBN = new BN(minStake.toString());
     });
 
+    it('makes the owner the ERC20 to Native Bridge', async () => {
```

Comment:
I suppose all tests would require sending some native coins to certain accounts. So maybe it's better to make `OWNER => bridge` outside the test suite, right after the ERC20 is deployed?

Reply:
`OWNER => bridge` is non-trivial.  I think it is worth testing in its own right.

Reply:
I mean that `makes the owner the ERC20 to Native Bridge` and `Owner issues initial balances to candidates` tests add some coins to candidates, but candidates will require coins for other tests too, so maybe we can move it outside of `0_staking` test and run before starting the test suit?

Reply:
@phahulin what about putting it in a `0_initial_balances` test?

Reply:
If we put it there, other tests would depend on it. What if I don't want to run the whole suite, but, say, only malicious test?

Reply:
@phahulin That is true.  I didn’t think of that.

Reply:
Fixed.